ushahidi/platform-client
# #3315 Fix translation on modal title for the Spanish Language

This pull request makes the following changes:
- Adds `translate` to span for Modal title
- Spanish language characters now appear as expected. 
- The title `Actualizar B&uacute;squeda` appears as `Actualizar Búsqueda`
- The title `Crear B&uacute;squeda Guardada` appears as `Crear Búsqueda Guardada`

Testing checklist:
- Select Spanish as Language on the Languages dropdown
- Go to sort & filters 
- Create a new filter.
- Click on the yellow button  `Guardar Búsqueda` 
- You will see `Crear Búsqueda Guardada`as modal title
- Update the created filter.
- Click on the yellow button `Actualizar Búsqueda`
- You will see `Actualizar Búsqueda` as modal title


Fixes ushahidi/platform#3315 .

Ping @ushahidi/platform
